composeApp/src/commonMain/kotlin/org/paystell/app/screens/authentication/ForgotPasswordScreen.kt

```
@@ -0,0 +1,251 @@
+                        delay(1500)
+
+                        // Simulate random network error (20% chance)
+                        if (Random.nextDouble() < 0.2) {
+                            throw Exception("Network error")
+                        }
+
+                        // Success path
+                        showSuccessDialog = true
+                        delay(3000) // Show success message briefly
+                        onResetRequestSent()
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)